pkg/csi/service/vanilla/controller.go

```diff
@@ -166,6 +168,16 @@ func (c *controller) Init(config *config.Config) error {
 		log.Errorf("Failed to create co agnostic interface. err=%v", err)
 		return err
 	}
+	if containerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIAuthCheck) {
```

Comment:
Where is this feature enabled? Should we not make changes in the yaml files?

Reply:
I enabled in my local test setup. Since this feature is not complete yet. Should we change the yaml file now?

Reply:
I added the following in the file

Reply:
You need to add this in the yaml file with the FSS turned off.